src/hypothesis/internal/conjecture/engine.py

```diff
@@ -1149,10 +1127,9 @@ def shrink(self):
             for c in corpus:
                 if sort_key(c) >= sort_key(self.last_data.buffer):
                     break
-                elif self.incorporate_new_buffer(c):
+                if self.incorporate_new_buffer(c):
                     break
-                else:
-                    self.settings.database.delete(self.secondary_key, c)
+                self.settings.database.delete(self.secondary_key, c)
```

Comment:
Minor: I think `if … elif … else` is more readable in this sort of scenario.

It makes it explicit that only one of these branches will be executed, without having to drop down a level and read the `break`s.

In this case it’s not so bad.